### ISSUE TITLE
ХФ 0.2 Невозможно удалить пост, если за него кто-то уже проголосовал

### ISSUE THREAD

**Issue by professor-fortran** [GolosChain/golos#266]
Почему-то нельзя удалить пост, после того, как его кто-то апвоутнул.
Если поставить флаг, то удалить пост можно
![1](https://user-images.githubusercontent.com/32443152/32679117-3f345eda-c676-11e7-98c2-6adfaf8b8f34.png)
Если никаких голосов нет, то удалить пост можно
![2](https://user-images.githubusercontent.com/32443152/32679138-52393410-c676-11e7-839b-324c7abfa9a7.png)
Если есть хотя бы один положительный голос (апвоут), то удалить пост нельзя
![0](https://user-images.githubusercontent.com/32443152/32679204-96441d78-c676-11e7-9e08-45a1d9c61aed.png)
P.S. Если это способ защиты инвестиций голосующих, то тогда надо убирать и возможность редактирования, т.к. в противном случае можно будет сделать так, что за пост проголосовало куча аккаунтов, а потом автор этого поста его изменил и какую-то дрянь написал вместо того контента, который был оценен голосующими.

**Comment by On1x** [comment]
Не согласен по поводу запрета редактирования. В блокчейне все равно останется все. Тут уже речь про то, что социальная вещь встроена в блокчейн.

**Comment by professor-fortran** [comment]
Я согласен, поэтому я за то, чтобы была и кнопка удалить, и кнопка редактировать.

**Comment by On1x** [comment]
@professor-fortran думаю дело в сложности механики учета. Юзер тратит свою энергию для голосования. Если пост удалять, то надо всю потраченную энергию других юзеров возвращать? А что с кураторскими? Выплаты? Не учитывать rshares удаленного поста?

**Comment by professor-fortran** [comment]
@On1x А как же удаление поста после флагов? Флаги ведь тоже как-то сказываются на тех, кто их ставит. Я так понимаю, это сделано специально, чтобы не флаговали все подряд. Если это так, то получается, что флаги откатить можно, а положительные голоса уже нельзя.

**Comment by On1x** [comment]
@professor-fortran я не проверял флаги - относительно системы это тоже голоса, но с отрицательным весом. Есть пример постов, которые удалены создателем после получения флага?

**Comment by Chiliec** [comment]
Возможно суть в распределении пула, ведь у постов только с флагами нечего делить.

**Comment by professor-fortran** [comment]
Мне кажется, что не стоит гадать, а нужен ответ разработчиков на вопрос: баг это или фича?

**Comment by litrbooh** [comment]
@kotbegemot 
ого, движу пошла. Неужели можно будет удалять посты?
